.github/workflows/run_tests.yml

```diff
@@ -146,52 +168,23 @@ jobs:
         run: |
-          sudo apt-get -y update && sudo apt-get -y install curl gnupg lsb-release software-properties-common ca-certificates && \
-          # Add latest redis to apt sources
-          sudo echo "deb [signed-by=/usr/share/keyrings/redis-archive-keyring.gpg] https://packages.redis.io/deb $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/redis.list && \
-          sudo curl -fsSL https://packages.redis.io/gpg | gpg --dearmor -o /usr/share/keyrings/redis-archive-keyring.gpg && \
-          # Add latest docker to apt sources
-          sudo mkdir -m 0755 -p /etc/apt/keyrings &&
-          sudo curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg &&
-          echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
-                $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null &&
-          # Install and update
-          sudo apt-get -y update && \
-          sudo apt-get -y install iputils-ping docker-ce docker-ce-cli containerd.io redis-tools=6:6.2.5-1rl1~focal1 redis-server=6:6.2.5-1rl1~focal1 &&
-          CONTAINER_NAME="redisai_$RANDOM" &&
-          sudo docker create --name $CONTAINER_NAME --rm redislabs/redisai:${{ matrix.rai_v }}-cpu-bionic && \
-          sudo mkdir -p /usr/lib/redis/modules/ &&
-          sudo docker cp $CONTAINER_NAME:/usr/lib/redis/modules/redisai.so /usr/lib/redis/modules &&
-          sudo docker cp $CONTAINER_NAME:/usr/lib/redis/modules/backends/ /usr/lib/redis/modules/ &&
+          make test-deps &&
           echo "LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$PWD/install/lib" >> $GITHUB_ENV &&
-          echo "REDISAI_CPU_INSTALL_PATH=/usr/lib/redis/modules/" >> $GITHUB_ENV &&
-          echo "SR_CICD_EXECUTION=True" >> $GITHUB_ENV &&
-          echo "REDISAI_MODULES=/usr/lib/redis/modules/redisai.so \
-                TF /usr/lib/redis/modules/backends/redisai_tensorflow/redisai_tensorflow.so \
-                TORCH /usr/lib/redis/modules/backends/redisai_torch/redisai_torch.so" >> $GITHUB_ENV
-
-      # Check the static build by compiling and linking examples
-      # Only supported on Linux; static builds are not currently available for MacOS
-      - name: Install MPI for parallel examples -- for GCC, we can just use the std install
-        if: contains(matrix.os, 'ubuntu') && !contains(matrix.compiler, 'intel')
-        run: sudo apt-get install -y mpich
-      - name: Test static build -- GNU compilers
-        if: contains(matrix.os, 'ubuntu') && !contains(matrix.compiler, 'intel')
-        run: make build-examples SR_LINK=Static SR_FORTRAN=ON
-      - name: Test static build -- intel compilers
-        if: contains(matrix.os, 'ubuntu') && contains(matrix.compiler, 'intel')
-        run: make build-example-serial SR_LINK=Static SR_FORTRAN=ON
+          echo "$PWD/third-party/redis/src" >> $GITHUB_PATH
```

Comment:
Why is this needed? The test drivers know how to find redis utils from the SR root

Reply:
Good point I'll track this down

Reply:
This was indeed not needed